src/brsTypes/components/RoSGNode.ts

```diff
@@ -317,6 +318,33 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
         return false;
     }
 
+    /**
+     * Calls the function specified on this node.
+     */
+    private callFunc = new Callable("callFunc", {
+        signature: {
+            args: [
+                new StdlibArgument("functionname", ValueKind.String),
+                new StdlibArgument("functionargs", ValueKind.Dynamic),
+            ],
+            returns: ValueKind.Dynamic,
+        },
+        impl: (interpreter: Interpreter, functionname: BrsString, functionargs: BrsType) => {
+            // We need to search the callee's environment for this function rather than the caller's.
+            let componentDef = interpreter.environment.nodeDefMap.get(this.nodeSubtype);
+            if (componentDef) {
+                return interpreter.inSubEnv(subInterpreter => {
+                    let functionToCall = subInterpreter.getCallableFunction(functionname.value);
+                    return functionToCall.call(subInterpreter, functionargs);
+                }, componentDef.environment);
+            } else {
+                throw new Error(
```

Comment:
This will stop execution of the script, right? That differ from the warning returned by RBI which doesn't stop execution.
```
=================================================================
Warning calling function in MyNode
no function interface specified for something
=================================================================
```